codespell_lib/data/dictionary.txt

```diff
@@ -7657,7 +7657,7 @@ irregardless->regardless
 irrelavent->irrelevant
 irrelevent->irrelevant
 irrelvant->irrelevant
-irreplacable->irreplaceable
+irreplacable->irreplaceable, irreparable,
```

Comment:
This does not look like a spelling error, but rather an English usage / word choice error. I don't think we should start adding those.